go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	cloud.google.com/go v0.71.0 // indirect
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.10.1
-	github.com/heimweh/go-pagerduty v0.0.0-20230404180624-b51b4562bcba
+	github.com/heimweh/go-pagerduty v0.0.0-20230412175655-9a15747f07a5
```

Comment:
I've updated to latest version: https://pkg.go.dev/github.com/heimweh/go-pagerduty@v0.0.0-20230412175655-9a15747f07a5

Reply:
This change is removed because I rebased off of the latest master branch.